packages/apidom-ls/test/fixtures/validation/oas/path-template-all-defined.yaml

```diff
@@ -0,0 +1,70 @@
+openapi: 3.0.0
+info:
+  title: Foo
+  version: 0.1.0
+components:
+  parameters:
+    test_id:
+      name: test_id
+      in: path
+      required: true
+      schema:
+        type: string
+        format: uuid
+        title: Test Id
+paths:
```

Comment:
The fixture is missing cases for `PathItem.parameters` - when parameters are defined only in `PathItem.paramters` or both in `PathItem.parameters` and `Operation.parameters`.